composer.json

```diff
@@ -78,6 +78,7 @@
         "upload-coverage": "coveralls -v"
     },
     "replace": {
+        "container-interop/container-interop": ">=1.2.0",
```

Comment:
Isn't this gonna be a problem if I have installed container-interop < 1.2.0 ?

Reply:
We already conflicted with versions less than that. :wink: